apiary.apib

```diff
@@ -0,0 +1,167 @@
+FORMAT: 1A
+HOST: http://api.floworky.com/api/v1
+
+# floworky
+
+Floworky is a workflow tool for productivity 
+
+## Items Table [/item]
+
+### List All Items [GET]
+
+You want to list all the items. It takes the user id email from the parameters.
+
++ Request (application/json)
+
+    + Headers
+    
+            Location: /list/:user_id/:email
+            Authorization: JSONWEBTOKEN
+
++ Response 200 (application/json)
+    
+    + Body
+
+            {
+              "items": [
+                {
+                  "id": 1,
+                  "title": "give dev $200",
+                  "description": "egiftcard",
+                  "completed": false,
+                  "parent_id": 0,
+                  "user_id": 8,
+                  "createdAt": "2016-10-20T00:31:56.430Z",
+                  "updatedAt": "2016-10-20T00:31:56.430Z"
+                }
+              ],
+              "tree": [
```

Comment:
We should either choose to send a tree, or send a list of items - I think it will confuse the API if we incorporate all possible views of the data into a single response.  I do envision a separate tree endpoint that takes a rootid, and just returns the tree rooted from that item.